hs/src/Reach/Connector/ETH_Solidity.hs

```diff
@@ -1202,6 +1202,8 @@ apiDef who ApiInfo{..} = do
           _ -> do
             tc_args <-
               case ai_msg_vs of
+                [DLVar _ _ (T_Tuple []) _] ->
+                  return $ [ "false" ]
```

Comment:
What is going on here?

Reply:
Found a small issue with the lifted API call. Usually we want to construct the type of the argument like `T22(..)` when calling the consensus function, but when the domain is empty, the function expects a boolean. So, this is outputs `bool` instead of `bool()`.

Reply:
You should make a note to have an example that demonstrates that problem.